UI/api-interface.cpp

```diff
@@ -320,6 +320,21 @@ struct OBSStudioAPI : obs_frontend_callbacks {
 		return os_atomic_load_bool(&recording_paused);
 	}
 
+	bool obs_frontend_recording_split_file(void) override
+	{
+		if (os_atomic_load_bool(&recording_active)) {
+			proc_handler_t *ph = obs_output_get_proc_handler(
+				main->outputHandler->fileOutput);
+			calldata *cd = calldata_create();
+			proc_handler_call(ph, "split_file", cd);
+			bool result = calldata_bool(cd, "split_file_enabled");
+			calldata_destroy(cd);
```

Comment:
```suggestion
			uint8_t stack[128];
			calldata cd;
			calldata_init_fixed(&cd, stack, sizeof(stack));
			proc_handler_call(ph, "split_file", &cd);
			bool result = calldata_bool(&cd, "split_file_enabled");
```
This is just a preference but allocating calldata on the stack might be better in the performance.